libtelemetry/src/main/java/com/mapbox/services/android/telemetry/TelemetryUtils.java

```diff
@@ -21,4 +25,14 @@ static String obtainUniversalUniqueIdentifier() {
     String universalUniqueIdentifier = UUID.randomUUID().toString();
     return universalUniqueIdentifier;
   }
+
+  public static String getApplicationIdentifier(Context context) {
```

Comment:
What about exposing a `createFullUserAgent` method instead? ☝️

Reply:
get/set nomenclature is hijacked by the Java language and should be only used when defining POJOs/DTOs.